### DISCUSSION TITLE
Compile For Aarch64 Alpine

### DISCUSSION THREAD

**Discussion by PublicVoidUpdate** [KRTirtho/spotube#514] [Q&A]
Hi! im looking for a spotify client to run on a PostMarketOS(Pretty much just alpine with an added repositiory) Linux Phone I saw you have an android APK and a Linux flatpak, so would it be possible to compile this for arm Alpine? and if so could you walk me through the general steps? 
Thank you,
-PublicVoidUpdate

**Comment by KRTirtho** [comment]
Build instructions can be found [here](https://github.com/KRTirtho/spotube/blob/master/CONTRIBUTION.md#your-first-code-contribution)
But these are for distro with libc but since alpine doesn't provide it out of the box I can't guarantee if it'd compile or not but u can give a try

**Reply by PublicVoidUpdate** [reply]
Thank you! Are there any environmental variables I need a set to work with aarch64 or will it auto detect?

**Reply by KRTirtho** [reply]
There's no env config. Flutter can automatically select arch
